client/src/mvc/ui/ui-select-content.js

```diff
@@ -425,15 +431,30 @@ const View = Backbone.View.extend({
         _.each(options, (items, src) => {
             _.each(items, (item) => {
                 self._patchValue(item);
+                console.log(item)
                 const current_src = item.src || src;
-                select_options[current_src].push({
+                if (this.model.attributes.tag) {
+                    if(item.tags.includes(this.model.attributes.tag)) {
+                        select_options[current_src].push({
```

Comment:
Recommend to use a flag here, so something like `addOption = true;`. This would avoid duplicating this code block.